src/applications/gi-sandbox/containers/SearchPage.jsx

```diff
@@ -80,6 +80,7 @@ export function SearchPage({
                   <LocationSearchForm />
                 </va-accordion-item>
               </va-accordion>
+              {search.name.count && <NameSearchResults />}
```

Comment:
This is in wrong place, it should be within the accordion for Name Search

Reply:
Apparently this is in right spot. 

But this has incorrect logic to determine if should be rendered.  There should be a check to see if the Name accordion is open as well

Reply:
```
document.querySelector("[header='Search by name']").getAttribute('open')
```